BookingsApi/BookingsApi.Infrastructure.Services/Publishers/HearingNotoficationEventForJudiciaryParticipantPublisher.cs

```diff
@@ -0,0 +1,30 @@
+﻿using BookingsApi.Common;
+using BookingsApi.Domain;
+using BookingsApi.Infrastructure.Services.IntegrationEvents;
+using BookingsApi.Infrastructure.Services.IntegrationEvents.Events;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BookingsApi.Infrastructure.Services.Publishers
+{
+    public class HearingNotoficationEventForJudiciaryParticipantPublisher : IPublishEvent
```

Comment:
typo (and in the filename) - `HearingNotificationEventForJudiciaryParticipantPublisher`

Reply:
done